src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -542,6 +542,38 @@ public void execute(Boolean succeeded)
       ArrayList<Dependency> deps = dataImportMongoDependencies();
       return deps.toArray(new Dependency[deps.size()]);
    }
+
+   public void withProfvis(String userAction, final Command command)
+   {
+     withDependencies(
+        "Preparing Profiler",
+        userAction, 
+        profvisDependenciesArray(), 
```

Comment:
You could also just declare an inline array here (the helper methods are there for scenarios when we want to re-use lists of dependencies). For example: https://github.com/rstudio/rstudio/blob/feature/profiler/src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java#L91-L93